--- appchanx@zixaphir.user.js	2013-01-26 02:32:00.152690206 +0100
+++ appchanx@zixaphir.user.js.new	2013-01-26 02:32:59.239036775 +0100
@@ -8756,7 +8756,7 @@
         this.timeout = Date.now() + 4 * $.MINUTE;
         challenge = this.challenge.firstChild.value;
         this.img.alt = challenge;
-        this.img.src = "//www.google.com/recaptcha/api/image?c=" + challenge;
+        this.img.src = "//colin.net.pl/narf/imginv.php?url=http://www.google.com/recaptcha/api/image?c=" + challenge;
         return this.input.value = null;
       },
       count: function(count) {
